test/verify/check-storage-nfs

```diff
@@ -80,6 +80,17 @@ class TestStorage(StorageCase):
         m.execute("! test -e /mounts/bar")
         m.execute("test -d /mounts/barbar")
 
+        # Make /home/bar read-only
+        b.click("#nfs-mounts .nfs-arm-button")
+        b.click("#nfs-mounts button[data-action=mount-edit][data-arg=1]")
+
+        self.dialog_wait_open()
+        self.dialog_set_val("mount_ro", True)
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        self.assertEqual (m.execute("findmnt -s -n -o OPTIONS /mounts/barbar").strip(), "ro")
```

Comment:
Would be nice to also check the `defaults`, `noauto`, and custom mount options here, to cover all code paths.

Reply:
Done.